library/core/src/cell.rs

```diff
@@ -226,7 +226,7 @@ use crate::ptr;
 /// assert_eq!(my_struct.special_field.get(), new_value);
 /// ```
 ///
-/// See the [module-level documentation](index.html) for more.
+/// See the [module-level documentation](super) for more.
```

Comment:
I think you want `self`, not `super`. `super` is the same as `crate` in this scope.